includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -580,6 +580,18 @@ public function init_hooks() {
 	public function get_title() {
 		$title = parent::get_title();
 
+		/**
+		 * Allows themes or other plugins to override whether the "rich" payment method label
+		 * (normally displayed on shortcode checkout/my account/pay-for-order pages) will be used.
+		 *
+		 * @since 8.6.0
+		 *
+		 * @param $use_plain_method_label boolean Whether the "plain" payment method label will be displayed.
+		 */
+		if ( apply_filters( 'wcpay_checkout_use_plain_method_label', false ) ) {
```

Comment:
I went with a `true`/`false` filter approach (rather than a filter on the `$title`), to prevent bad actors from having an easy way into customizing the payment method title.